packages/site/package.json

```diff
@@ -23,9 +23,9 @@
     "e2e:codegen": "npx playwright codegen localhost:3000",
     "start": "next start",
     "gen:snapshot": "yarn mosaic build --out snapshots --name latest --config mosaic.config.js",
-    "serve:snapshot:file": "yarn cross-env MOSAIC_MODE='snapshot-file' concurrently --kill-others 'yarn dev'",
-    "serve:snapshot:s3": "yarn cross-env MOSAIC_MODE='snapshot-s3' concurrently --kill-others 'yarn dev'",
-    "serve": "concurrently --kill-others 'yarn dev' 'yarn mosaic serve -c ./mosaic.config.js' -p 8080"
+    "serve:snapshot:file": "yarn cross-env MOSAIC_MODE='snapshot-file' NODE_ENV='local' concurrently --kill-others 'yarn dev'",
```

Comment:
Do you still need these changes?

Reply:
yeah I think so otherwise NODE_ENV is undefined and it doesn't load env.local

Reply:
Have you tried without? dotenv-load should handle this

Reply:
ohhh it does that's cool, will remove